models/ed/inst/ED2IN.rgit_istfer

```diff
@@ -1292,7 +1292,7 @@ $ED_NL
    !---------------------------------------------------------------------------------------!
    ! IOPTINPT -- Optimization configuration. (Currently not used)                          !
    !---------------------------------------------------------------------------------------!
-   NL%IOPTINPT = ''
+   ! NL%IOPTINPT = ''
```

Comment:
I would just delete this block (including the comment above it) so it's not confusing. I think this has been dropped from the ED2 mainline.

Reply:
You'll want to verify that it has been completely removed from the ED2 mainline, not just that it's currently not used, as FORTRAN namelists are not flexible to missing values the way that XML and JSON are -- if a tag is missing in the namelist it will typically throw a cryptic error by misparsing the next or previous tag.

Reply:
Considering I've had multiple ED runs fail on me recently because the tag _was_ in there when ED wasn't expecting it, I think it's safe to say it's gone.

Reply:
Yes, it has been dropped from the mainline (it actually happened in my PR) I was keeping it just in case I need to go back and do some tests with older ED2 versions, but I can also delete it